esmvaltool/diag_scripts/validation.py

```diff
@@ -0,0 +1,274 @@
+"""
+Validation Diagnostic
+
+This diagnostic uses two datasets (control and experiment),
+applies operations on their data, and plots one against the other.
+It can optionally use a number of OBS, OBS4MIPS datasets.
+"""
+
+import os
+import logging
+
+import numpy as np
+
+import iris
+import iris.analysis.maths as imath
+import iris.quickplot as qplt
+
+from esmvaltool.diag_scripts.shared import (group_metadata, run_diagnostic,
+                                            select_metadata)
+from esmvaltool.preprocessor._area_pp import area_slice
+
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt  # noqa
+
+
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def plot_contour(cube, plt_title, file_name):
+    """Plot a contour with iris.quickplot (qplot)"""
+    qplt.contourf(cube, cmap='RdYlBu_r', bbox_inches='tight')
+    plt.title(plt_title)
+    plt.gca().coastlines()
+    plt.tight_layout()
+    plt.savefig(file_name)
+    plt.close()
+
+
+def plot_latlon_cubes(cube_1, cube_2, cfg, data_names, obs_name=None):
+    """
+    Plot different things
```

Comment:
I wonder what those things are .. ;-)

Reply:
yes, it is self-explanatory - plot different things :grin: sorry, bud, me being lezy, I'll clean up, fix the duplication bits and add some nicely explained docstrings